content/version/1/4/code-of-conduct.hi.md

```diff
@@ -3,76 +3,48 @@ version = "1.4"
-उन्मुखीकरण।
+खुले और सौहार्दपूर्ण वातावरण को बढ़ावा देने के हित में, हम, योगदानकर्ता एवं अनुरक्षक के रूप में, परियोजना और समुदाय में हमारी भागीदारी को उम्र, शरीर के आकार, विकलांगता, जाति, लिंग पहचान और उसकी अभिव्यक्ति, अनुभव का स्तर, राष्ट्रीयता, व्यक्तिगत रूप-देखाव, कुल, धर्म अथवा यौन पहचान और उसके अभिविन्यास, पर ध्यान दिए बगैर सभी के लिए उत्पीड़न-मुक्त अनुभव बनाने की प्रतिज्ञा करते हैं ।
 
-हमारे मानकों ##
+## हमारे मानक
 
-व्यवहार है कि एक सकारात्मक माहौल बनाने के लिए योगदान के उदाहरण
-शामिल:
+सकारात्मक माहौल बनाने वाले व्यवहारों के उदाहरणों में शामिल है :
 
-* स्वागत करते हैं और समावेशी भाषा का प्रयोग
-* भिन्न दृष्टिकोण और अनुभवों का सम्मान किया जा रहा
-* सुन्दरता रचनात्मक आलोचना को स्वीकार
-* क्या समुदाय के लिए सबसे अच्छा है पर ध्यान केंद्रित
-* अन्य समुदाय के सदस्यों के प्रति सहानुभूति दिखा
+* सौहार्दपूर्ण, अभिनंदनीय और समावेशी भाषा का प्रयोग
+* भिन्न दृष्टिकोण और अनुभवों का सम्मान करना
+* योग्य रूप से आलोचना को स्वीकारना
+* समुदाय के लिए जो श्रेष्ठ है उस पर ध्यान केंद्रित करना
+* समुदाय के अन्य सदस्यों के प्रति सहानुभूति दिखाना
 
-प्रतिभागियों द्वारा अस्वीकार्य व्यवहार के उदाहरण हैं:
+प्रतिभागियों द्वारा अस्वीकार्य व्यवहार के उदाहरणों में शामिल है :
 
-* एक कामुक भाषा या कल्पना और अनिष्ट यौन ध्यान या का उपयोग
-अग्रिमों
-* Trolling, अपमान / अपमानजनक टिप्पणी, और व्यक्तिगत या राजनीतिक हमलों
+* कामुक भाषा अथवा चित्र या अनभिलषित यौन ध्यान अथवा कोशिशें
+* दुर्व्यवहार, अपमानजनक/लज्जाजनक टिप्पणी अथवा व्यक्तिगत या राजनीतिक हमला
 * सार्वजनिक या निजी उत्पीड़न
-* प्रकाशन दूसरों के इस तरह के एक भौतिक या इलेक्ट्रॉनिक रूप में निजी जानकारी,
-  पता, स्पष्ट अनुमति के बिना
-* अन्य आचरण जो यथोचित एक में अनुचित माना जा सकता है
-  पेशेवर सेटिंग
+* दूसरों की निजी जानकारी, जैसे वास्तविक अथवा इलेक्ट्रॉनिक पता, बिना उनके अनुमति के प्रकाशित करना
+* अन्य आचरण, जिसे व्यवसायी परिस्थिति में, अनुचित ठहराया जा सकता है
 
 ## हमारा उत्तरदायित्व
 
-परियोजना देखरेख स्वीकार्य मानकों के स्पष्ट करने के लिए जिम्मेदार हैं
-व्यवहार और में उचित और निष्पक्ष सुधारात्मक कार्रवाई करने के लिए उम्मीद कर रहे हैं
-अस्वीकार्य व्यवहार का कोई उदाहरण के जवाब।
+परियोजना अनुरक्षकों पर व्यवहार के स्वीकार मानकों के स्पष्टीकरण की जिम्मेदारी है। किसी भी अस्वीकार्य व्यवहार के जवाब में उचित और निष्पक्ष सुधारात्मक कार्रवाई करने की अपेक्षा भी अनुरक्षकों से की जाती है।
 
-परियोजना देखरेख सही और जिम्मेदारी को दूर संपादित करें, या करने के लिए है
-टिप्पणियाँ, करता है, कोड, विकी संपादन, मुद्दों, और अन्य योगदान अस्वीकार
-कि इस आचार संहिता के लिए गठबंधन नहीं कर रहे हैं, या अस्थायी रूप से प्रतिबंध लगाने के लिए या
-स्थायी रूप से अन्य व्यवहार है कि वे अनुचित समझे के लिए किसी भी योगदानकर्ता,
-धमकी, आक्रामक, या हानिकारक है।
+परियोजना अनुरक्षकों को टिप्पणियों, कमिट, कोड, विकी संपादन, मुद्दों और अन्य योगदानों, जो इस आचार संहिता के मुताबिक नहीं है, को हटाने, सम्पादित करने या अस्वीकार करने के साथ-साथ किसी भी योगदानकर्ता, जिसका व्यवहार अनुचित, धमकीभरा, आक्रामक या नुकसानदेय समझा जा सकता है, को अस्थायी या स्थायीरूप से प्रतिबंधित करने की जिम्मेदारी और ऐसा करने का अधिकार है।
```

Comment:
Some restructuring for clarity:

परियोजना अनुरक्षकों का अधिकार है ऐसी टिप्पणियों, कमिट, कोड, विकी संपादन, मुद्दों और अन्य योगदानों को हटाने, सम्पादित करने या अस्वीकार करने का, जो इस आचार संहिता के मुताबिक नहीं हैं। साथ-साथ, उनकी जिम्मेदारी और अधिकार है किसी भी ऐसे योगदानकर्ता को अस्थायी या स्थायीरूप से प्रतिबंधित करने की, जिनका व्यवहार अनुचित, धमकीभरा, आक्रामक या नुकसानदेय समझा जा सकता है ।